core/statix.multilang/src/main/java/mb/statix/multilang/spec/ASpecBuilder.java

```diff
@@ -0,0 +1,57 @@
+package mb.statix.multilang.spec;
+
+import mb.statix.spec.Spec;
+import org.immutables.value.Value;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Value.Immutable
+public interface ASpecBuilder {
```

Comment:
I see that you're using `SpecBuilder` everywhere else, which is the implementation of this class (generated by immutables, I think?). Is there a reason why you are using the class everywhere instead of this in terface? It would make more sense to call this interface `SpecBuilder`, use that everywhere, and have the implementation be called `ImmutableSpecBuilder`?

Reply:
This is the Statix convention, which I adopted because the `ImmutableXXX` names are quite long to read. But thinking about it it would of course be better to make it consistent within Spoofax 3, so I'll rework this.